jupyter_book/config.py

```diff
@@ -128,6 +128,14 @@ def get_final_config(
     # and completely override any defaults (sphinx and yaml)
     sphinx_config.update(user_yaml_update)
 
+    # This is to deal with a special case, where the override needs to be applied after
+    # the sphinx app is initialised (since the default is a function)
+    # TODO I'm not sure if there is a better way to deal with this?
+    config_meta = {
```

Comment:
why has this been moved to above the sphinx_config update?

Reply:
@chrisjsewell from memory it was to be in front of:

```python
    # finally merge in CLI configuration
    _recursive_update(sphinx_config, cli_config or {})
```

as I was getting a key error on `cli_config.pop("latex_individualpages")` when it was placed below but I can check this again.

Reply:
confirmed -- it needs to be above this line to access the default value from `cli_config` when `--individualpages` isn't specified on the `command line`